interfaces/udev/udev.go

```diff
@@ -24,6 +24,22 @@ import (
 	"os/exec"
 )
 
+// udevadmTrigger runs "udevadm trigger" but ignores an non-zero exit codes.
+// udevadm only started reporting errors in systemd 248 and in order to
+// work correctly in LXD these errors need to be ignored. See
+// https://github.com/lxc/lxd/issues/9526 for some more background.
```

Comment:
Maybe link to https://github.com/systemd/systemd/pull/18559 or better the actual change in https://github.com/systemd/systemd/pull/18684 as well